web/styles/settings.css

```diff
@@ -1490,6 +1490,41 @@ $option_title_width: 180px;
     }
 }
 
+.dropdown-list-widget .dropdown-list-body,
+#gear-menu .dropdown-menu {
+    /* In bootstrap v2.3.2, we strictly need
+        .dropdown-menu > li > a as the order of the elements
+        for the bootstrap style to be applied.
+        Since we use a combination of dropdown + simplebar here,
+        the structure cannot be possible since simplebar inserts
+        elements of its own, so we copy over that style
+        from bootstrap to here. */
+    & li a {
+        display: block;
+        padding: 3px 20px;
+        clear: both;
+        font-weight: normal;
+        line-height: 20px;
+        color: hsl(0deg 0% 20%);
+        white-space: nowrap;
+
+        &:hover {
+            color: hsl(0deg 0% 100%);
+            text-decoration: none;
+            background-color: hsl(200deg 100% 38%);
+            background-image: linear-gradient(
+                to bottom,
+                hsl(200deg 100% 40%),
+                hsl(200deg 100% 35%)
+            );
```

Comment:
<img width="202" alt="image" src="https://github.com/zulip/zulip/assets/25124304/c99f7dd6-dbc9-4a2c-93c0-9b5bcf54825d">

Dark theme is broken due to this change.

Reply:
Also, our fancy focus highlights are gone.
<img width="202" alt="image" src="https://github.com/zulip/zulip/assets/25124304/76135514-62a8-46b3-86ad-ef9624e24ff6">

Reply:
Fixed!